interfaces/console/src/app/console/sites/[id]/page.tsx

```diff
@@ -302,66 +318,105 @@ const Page: React.FC<SiteDetailsProps> = ({ params }) => {
     },
   });
 
-  const [getMetricStat] = useGetMetricsStatLazyQuery({
-    client: subscriptionClient,
-    fetchPolicy: 'network-only',
-    onCompleted: (data) => {
-      if (data.getMetricsStat.metrics.length > 0) {
-        data.getMetricsStat.metrics.forEach((m) => {
-          if (m.type === 'unit_uptime') {
-            setNodeUptime(m.value);
+  const [getMetricStat, { loading: statLoading, variables: statVar }] =
+    useGetMetricsStatLazyQuery({
+      client: subscriptionClient,
+      fetchPolicy: 'network-only',
+      onCompleted: (data) => {
+        if (data.getMetricsStat.metrics.length > 0) {
+          setNodeUptimes((prev) => {
+            const newUptimes = { ...prev };
+            data.getMetricsStat.metrics.forEach((m) => {
+              if (m.type === NODE_UPTIME_KEY) {
+                newUptimes[m.nodeId] = m.value;
+              }
+            });
+            return newUptimes;
+          });
+
+          if (statVar && statVar.data && statVar.data.from) {
+            const from = statVar.data.from;
+            const sKey = `stat-${user.orgName}-${user.id}-${Stats_Type.AllNode}-${data.getMetricsStat.metrics[0].nodeId}-${from}`;
+            MetricStatSubscription({
+              key: sKey,
+              nodeId: data.getMetricsStat.metrics[0].nodeId,
+              userId: user.id,
+              url: env.METRIC_URL,
+              orgName: user.orgName,
+              type: Stats_Type.AllNode,
```

Comment:
Why we are fetching `AllNode` stats on site detail page. Shouldn't that be `SiteStats`

Reply:
@salman-codewar We need node uptime on site page.